frontend/src/types.ts

```diff
@@ -41,6 +41,8 @@ export interface UserType {
     organization: OrganizationType | null
     team: TeamBasicType | null
     organizations: OrganizationBasicType[]
+    realm: 'cloud' | 'hosted' | 'hosted-clickhouse'
```

Comment:
This is an instance-based property, not a user-based property, I don't think we should send this in the user payload at all. See also comment about sending this in the deprecated endpoint.